README.md

```diff
@@ -37,7 +37,7 @@ The following tasks are no longer being developed and will be deprecated in a fu
 
 | Task Name | Alternative |
 |-----------|-------------|
-| st0317a_clean_cert | User [certificate clean](https://www.puppet.com/docs/puppet/7/server/http_certificate_clean) API to remove certifications |
+| st0317a_clean_cert | Use [certificate clean](https://www.puppet.com/docs/puppet/7/server/http_certificate_clean) API to remove certifications | 
```

Comment:
Is this a copy-paste mistake?  This api won't clear any file sync locks.

Reply:
Yeah that was a mistake, I'll remove it

Reply:
I noticed the spell check and realized it needed corrected